packages/compass-connections/src/hooks/use-connection-repository.ts

```diff
@@ -89,18 +90,18 @@ export function useConnectionRepository(): ConnectionRepository {
       );
 
       setFavoriteConnections((prevList) => {
-        if (areConnectionsEqual(prevList, favoriteConnections)) {
+        if (areConnectionsEqual(prevList, newFavoriteConnections)) {
           return prevList;
         } else {
-          return favoriteConnections;
+          return newFavoriteConnections;
         }
       });
 
       setNonFavoriteConnections((prevList) => {
-        if (areConnectionsEqual(prevList, nonFavoriteConnections)) {
+        if (areConnectionsEqual(prevList, newNonFavoriteConnections)) {
           return prevList;
         } else {
-          return nonFavoriteConnections;
+          return newNonFavoriteConnections;
```

Comment:
no change here, just touched this file while exploring some other options. there are higher scope consts with the name `favoriteConnections` and `nonFavoriteConnections`.